demo/src/scenes/rafxmark_scene.rs

```diff
@@ -95,7 +99,7 @@ impl RafxmarkScene {
         let sprite_spawner_system = SystemBuilder::new("sprite_spawner")
             .read_resource::<TimeState>()
             .read_resource::<VisibilityRegion>()
-            .write_resource::<SpriteRenderNodeSet>()
+            .write_resource::<SpriteRenderObjectRegistry>()
```

Comment:
I generally use the word "registry" with things that get registered and stay forever, or at least a long time. Like the RenderRegistry where we permanently register features/phases. Think RenderObjectSet could be more clear that it's a set of objects that changes regularly. (And it's shorter)

Reply:
(I can do this as a follow-up change if you like)

Reply:
Renamed.